mtags/src/main/scala-3/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -0,0 +1,169 @@
+package scala.meta.internal.pc
+
+import scala.collection.mutable
+import scala.meta.internal.pc.IdentifierComparator
+
+import dotty.tools.dotc.core.Contexts._
+import dotty.tools.dotc.core.Flags
+import dotty.tools.dotc.core.Flags._
+import dotty.tools.dotc.core.Names._
+import dotty.tools.dotc.core.Symbols._
+import dotty.tools.dotc.core.StdNames._
+import dotty.tools.dotc.core.Types._
+import dotty.tools.dotc.interactive.Completion
+import dotty.tools.dotc.transform.SymUtils._
+import dotty.tools.dotc.util.Spans
+import dotty.tools.dotc.util.SrcPos
+import dotty.tools.dotc.util.{NameTransformer, NoSourcePosition, SourcePosition}
+
+class CompletionProvider(
+    pos: SourcePosition,
+    ctx: Context
+) {
+    implicit val context: Context = ctx
+      
+    def completions() : List[Completion]
+        = Completion.completions(pos)._2.filterInteresting.sorted(completionOrdering(ctx, pos))
```

Comment:
```suggestion
        = {
        val (_, completions) = Completion.completions(pos) 
        completions.filterInteresting.sorted(completionOrdering(ctx, pos))
      }
```
I think looks a bit better.

Reply:
Agree.